ph5/clients/tests/test_ph5availability.py

```diff
@@ -73,36 +63,25 @@ def checkFieldsMatch(fieldNames, fieldsList, dictList):
     return True
 
 
-class TestPH5Availability(unittest.TestCase):
-
+class TestPH5Availability(LogTestCase, TempDirTestCase):
     def setUp(self):
         """
         setup for tests
```

Comment:
These kind of comments add no information. The function is already called `setup`.
Comments should add information / prevent code scanning to understand what the function does at a high level.
e.g.
creates and iniailtizes ph5 and availability object
creates a handle self.ph5test_path to the location of the ph5 test fixtures.
called before each test is run

Reply:
I agree. You can see this kind of comment everywhere in unittests. I will remove the ones in tests that I have changed.